Glimps/main.py

```diff
@@ -1,10 +1,30 @@
+from glimps.models import GLIMPSModule
 from sekoia_automation.module import Module
 
-from glimps import RetrieveAnalysis, SearchPreviousAnalysis, SubmitFileToBeAnalysed
+from glimps.export_action import ExportSubmission
+from glimps.search_analysis_by_sha256_action import SearchPreviousAnalysis
+from glimps.submit_file_to_be_analysed_action import WaitForFile
+from glimps.submit_file_to_be_analysed_action import SubmitFile
+from glimps.get_status_action import GetStatus
+from glimps.retrieve_analysis_action import RetrieveAnalysis
+from glimps.deprecated import (
+    DeprecatedRetrieveAnalysis,
+    DeprecatedSearchPreviousAnalysis,
+    DeprecatedSubmitFileToBeAnalysed,
+)
 
 if __name__ == "__main__":
-    module = Module()
-    module.register(RetrieveAnalysis, "get-results/{uuid}")
-    module.register(SearchPreviousAnalysis, "get-search/{sha256}")
-    module.register(SubmitFileToBeAnalysed, "post-submit")
+    deprecated_module = Module()
+    deprecated_module.register(DeprecatedRetrieveAnalysis)
+    deprecated_module.register(DeprecatedSearchPreviousAnalysis)
+    deprecated_module.register(DeprecatedSubmitFileToBeAnalysed)
+    deprecated_module.run()
+
+    module = GLIMPSModule()
+    module.register(ExportSubmission)
+    module.register(SearchPreviousAnalysis)
+    module.register(WaitForFile)
+    module.register(SubmitFile)
+    module.register(GetStatus)
+    module.register(RetrieveAnalysis)
```

Comment:
Only one module can be defined at a time.
The `register` method requires two arguments: the python class that implement the action and the command name that must be the same as the `docker_parameters` field in the action definition.
```suggestion
    module = GLIMPSModule()
    module.register(ExportSubmission, "ExportSubmission")
    module.register(SearchPreviousAnalysis, "SearchPreviousAnalysis")
    module.register(WaitForFile, "WaitForFile")
    module.register(SubmitFile, "SubmitFile")
    module.register(GetStatus, "GetStatus")
    module.register(RetrieveAnalysis, "RetrieveAnalysis")
    
    # register deprecated actions
    deprecated_module.register(DeprecatedRetrieveAnalysis, "get-results/{uuid}")
    deprecated_module.register(DeprecatedSearchPreviousAnalysis, "get-search/{sha256}")
    deprecated_module.register(DeprecatedSubmitFileToBeAnalysed, "post-submit")
```